### PR TITLE
Ks/#1193 modify max pull cnt

### PR DESCRIPTION
**Pull request by KSchopmeyer**
See commit for info

This fixes a bug in pywbemcli.py with handling of --pull_max_cnt option in interactive mode

Update by Andy: PR #1190 is merged.

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -29,6 +29,8 @@ Released: not yet
 * Fix issue where the instance shrub --fullpath option was not displaying the
   paths. (see issue #1180)
 
+* Fixed new formatting issues raised by flake8 5.0.
```
Comment (andy-maier) [inline docs/changes.rst:32]: The change log entry does not match at all with the changes in the PR.

Also, I cannot relate the PR title with the changes in the PR.

Can you please rebase this PR to master, and make its PR title, commit message and change log consistent with the actual changes?

**Review by andy-maier — CHANGES_REQUESTED** [review]
See comment

File: docs/changes.rst (hunk 1 above)
Comment (KSchopmeyer) [reply docs/changes.rst:32]: 1. Entry made in change log
2. The changes in the PR are
   * one line change to pywbemcli.py
   * Add tests
 3. Rebased to master

**Review by andy-maier — APPROVED** [review]

**Comment by andy-maier** [comment]
I rebased the PR to resolve the merge conflict on docs/changes.rst.

**Pull request merged**